toolkit/pythonit_toolkit/api/service_client.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+import httpx
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+from pythonit_toolkit.pastaporto.tokens import generate_service_to_service_token
+
+
+@dataclass
+class ServiceResponse:
+    errors: Optional[Dict[str, Any]]
+    data: Optional[Dict[str, Any]]
+
+
+class ServiceClient:
+    def __init__(
+        self,
+        url: str,
+        issuer: str,
+        audience: str,
+        jwt_secret: str,
+    ):
+        self.url = url
+        self.issuer = issuer
+        self.audience = audience
+        self.jwt_secret = jwt_secret
+
+        for arg in ("url", "issuer", "audience", "jwt_secret"):
+            if not getattr(self, arg):
+                raise ValueError(f"Argument '{arg}' can't be empty")
+
+    async def execute(
```

Comment:
can we use async in the pycon backend since it is django? 🤔

Reply:
actually maybe it doesn't matter if it is django or not, but how we run it on the lambda, if we run the ASGI version? I don't remember what was needed to run django under to have async working 🤔